portfolio/src/main/webapp/index.html

```diff
@@ -9,22 +9,23 @@
   <body>
     <div id="content">
       <ul id="navbar">
-        <li class="nav-button current" ><a>Home</a></li>
-        <li class="nav-button"><a href="introduction.html">Introduction</a></li>
-        <li class="nav-button"><a>Projects</a></li>
-        <li class="nav-button"><a>Gallery</a></li>
+        <li class="nav-button current" ><a class="nav-link">Home</a></li>
+        <li class="nav-button"><a href="introduction.html" class="nav-link">Introduction</a></li>
+        <li class="nav-button"><a href="projects.html" class="nav-link">Projects</a></li>
+        <li class="nav-button"><a href="gallery.html" class="nav-link">Gallery</a></li>
```

Comment:
The way I would recommend you do this is add the placeholders for the nav bar but wait to add the href attribute until the PR that adds that page.

Reply:
Can you also add some alt text to all your links?